lib/providers/search/search.dart

```diff
@@ -9,24 +9,24 @@ import '../user_auth/authenticate.dart';
 import '../../models/boards/post.dart';
 
 class Search extends ChangeNotifier with Toast {
-  Authenticate _authProvider;
+  late Authenticate _authProvider;
```

Comment:
이 부분도 
```suggestion
  Authenticate _authProvider;
 Search({required this._authProvider})
```
가 가능하긴 한데 사실 어떤 게 더 좋은 건지는 모르겠습니다. 비슷할 것 같네요